### PR TITLE
Tc 271 add new regulation to group

### PR DESCRIPTION
**Pull request by kevincarrogan**

### CODE REVIEW

**Comment by kevincarrogan** [comment]
I'm likely to do further re-factoring of this later when I can start seeing more commonalities between the views that I'll be producing (a lot of them will be very similar to this one).

File: dit_helpdesk/cms/views.py [hunk 1]
```diff
@@ -58,3 +60,36 @@ def get_queryset(self):
...
+        regulation_form_class = self.get_regulation_form_class()
+        regulation_form = regulation_form_class(regulation_group, request.POST)
+        if regulation_form.is_valid():
+            regulation_form.save()
+            return redirect(
+                "cms:regulation-group-detail",
+                pk=regulation_group.pk,
+            )
+
+        self.object = self.get_object()
```
Comment (andrzejlichaczewski) [inline dit_helpdesk/cms/views.py:93]: Why is it needed to attach to self?

Comment (kevincarrogan) [reply dit_helpdesk/cms/views.py:93]: Because internally the DetailView expects that when we're rendering the page again.

Comment (kevincarrogan) [reply dit_helpdesk/cms/views.py:93]: Specifically https://github.com/django/django/blob/f87b0ecd37e64e7a019d472de37d0789a8790f1f/django/views/generic/detail.py#L94

Comment (andrzejlichaczewski) [reply dit_helpdesk/cms/views.py:93]: Cool 👍 . Yeah I'll have to remind myself how these work, looks like what you have now may be the most practical solution for now.

**Pull request merged**